main-labs/sdn-openflow/pox/01-pox-controller/README.md

```diff
@@ -32,11 +32,13 @@ ovs-ofctl dump-flows s1
 
 This happens since the switch acts as a L2 learning switch by the default, but does not fill the table.
 
-To enable the `forwarding.l2_learning` POX component and see the flow rules in the switch, go into the `controller`
-terminal and type:
+To enable the `forwarding.l2_learning` POX component and see the flow rules in the switch, launching ```kathara connect controller```.
+
+Launch in the root@controller:
```

Comment:
I prefer the old version, since you should already have the controller terminal opened from the `kathara lstart` command.